packages/sources/css/src/components/actions/button/src/index.css

```diff
@@ -259,7 +259,7 @@
 }
 
 .vtmn-btn_size--medium.vtmn-btn--icon-alone span[class^='vtmx-'] {
-  font-size: var(--vtmn-typo_text-2-font-size);
+  font-size: var(--vtmn-typo_title-5-font-size);
```

Comment:
Semantically, it's weird to use a "title" design token in a button.
Can you check with @N0t-Simon, Sabrina or Marie? Thanks.

Reply:
Semantically, I agree with you but this is the token for the wanted size. Otherwise I would have to set the size in px.

Reply:
PS: it is for the icon, not the label text

Reply:
Yes, it's for icons (the button "icon-alone"). 
I will let you decide on the naming convention of the token; is it a new one? 
In any case, it needs to be larger than the "classic body text."

Reply:
@lauthieb Do you mean that we should create specific design tokens for icon sizes ?

Reply:
Ok, problem solved I think. As the icons must not change size according to the device screen size. We have to set icon size in px, not use typography design token.